app/assets/stylesheets/components/lesson/_navigation.scss

```diff
@@ -0,0 +1,50 @@
+.lesson-navigation {
+  margin-top: 300px;
+
+  &__element {
+    margin-bottom: 32px;
+    display: flex;
+    align-items: center;
+  }
+
+  &__marker {
+    width: 16px;
+    height: 16px;
+    border-radius: 50%;
+
+    &--primary {
+      background: $primary;
+    }
+
+    &--coffee {
+      background: $coffee;
+    }
+
+    &--whitish-grey {
+      background: $whitish-grey;
+    }
+
+    &--light-grey {
+      background: $light-grey;
+    }
+
+    &--greyish-brown {
+      background: $text-primary;
+    }
+  }
+
+  .line {
```

Comment:
Can we put this css declaration at the bottom of the file please? I think having the `&__title` element with the in the along with the rest of the `lesson-navigation` elements will make it more readable.

Also does the line class need to be nested under the `lesson-navigation` class here? if it doesn't can we take it out of the scope of lesson-navigation please.